oak-search/src/main/java/org/apache/jackrabbit/oak/plugins/index/search/spi/editor/FulltextDocumentMaker.java

```diff
@@ -362,10 +371,15 @@ private boolean addTypedOrderedFields(D doc,
                                           PropertyDefinition pd) {
         // Ignore and warn if property multi-valued as not supported
         if (property.getType().isArray()) {
-            log.warn(
-                    "[{}] Ignoring ordered property {} of type {} for path {} as multivalued ordered property not supported",
-                    getIndexName(), pname,
-                    Type.fromTag(property.getType().tag(), true), path);
+            // Log the warning if the counter goes beyond 1000.
+            // We could miss certain paths being logged here since the DocumentMaker is created for each node state for each index.
+            // But ideally a warning with the property in question should suffice.
+            if (!throttleWarnLogs || (throttleWarnLogs && WARN_LOG_COUNTER_MV_ORDERED_PROPERTY.incrementAndGet() > 1000)) {
```

Comment:
this can be simplified
```suggestion
            if (!throttleWarnLogs || WARN_LOG_COUNTER_MV_ORDERED_PROPERTY.incrementAndGet() > 1000) {
```